financial-connections/src/main/java/com/stripe/android/financialconnections/features/accountpicker/AccountPickerViewModel.kt

```diff
@@ -76,7 +77,13 @@ internal class AccountPickerViewModel @Inject constructor(
                     activeAuthSession.skipAccountSelection == true,
                 accounts = accounts,
                 selectionMode = if (manifest.singleAccount) SelectionMode.RADIO else SelectionMode.CHECKBOXES,
-                accessibleData = AccessibleDataCalloutModel.fromManifest(manifest),
+                accessibleData = AccessibleDataCalloutModel(
+                    businessName = manifest.businessName,
+                    permissions = manifest.permissions,
+                    isNetworking = false,
```

Comment:
Do we want to hardcode to false here?

Reply:
The regular Account Picker screen is not part of the networking flow, in this case the accessible data callout does not have to mention anything about networking.